app/models/miq_server.rb

```diff
@@ -204,7 +204,7 @@ def self.seed
   def self.start
     validate_database
 
-    EvmDatabase.seed_primordial
+    EvmDatabase.seed_primordial unless ENV['PRIMORDIAL'] == 'false' && MiqDatabase.count > 0
```

Comment:
I don't like how these things are in non-obvious places.

If we do this, I'd rather this live in `EvmDatabase` 

```ruby
class EvmDatabase
  def self.seed_primordial
    if ENV['SOME_VARIABLE']
      STDERR.puts "SKIPPING primordial seeding via SOME_VARIABLE, unset and re-run seed_primordial if this causes problems.
      return
    end
    ...
  end
end